api/controllers/apiDocs.js

```diff
@@ -0,0 +1,7 @@
+const swaggerDoc = require('fs').readFileSync(require.resolve('../swagger/swagger.yaml'), 'utf8');
+const yaml = require('node-yaml');
+
+module.exports.apiDocs = (req, res, next) => {
+  res.json(yaml.parse(swaggerDoc));
```

Comment:
I'm not sure why `node-yaml` exists, as it appears to wrap `js-yaml`, which does exactly what we want anyway. Should use the `safeLoad` method from that package instead.